plugins/lsp/language-servers/default.nix

```
@@ -421,6 +421,11 @@ with lib; let
         '';
       };
     }
+    {
+      name = "pest_ls";
+      description = "Enable pest-language-server, for pest";
+      package = pkgs.pest-ide-tools;
+    }
```

```suggestion
    {
      name = "pest-ls";
      description = "Enable pest-language-server, for pest";
      package = pkgs.pest-ide-tools;
      serverName = "pest_ls";
    }
```

We would adopt "nix-style" in this case.